src/aspire/utils/filters.py

```diff
@@ -47,17 +47,11 @@ def evaluate(self, omega):
     def _evaluate(self, omega):
         raise NotImplementedError('Subclasses should implement this method')
 
-    def radialize(self):
-        raise NotImplementedError('Subclasses should implement this method')
-
     def fb_mat(self, fbasis):
```

Comment:
Why have this function? It seems that everywhere else we use `filter_to_fb_mat`.

Reply:
Change to use the public function of Filter class for all code.